clusters/app.ci/supplemental-ci-images/telco-bastion-cs9.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
+kind: List
+items:
+- kind: ImageStream
+  apiVersion: image.openshift.io/v1
+  metadata:
+    namespace: ci
+    name: telco-bastion-cs9
+- kind: BuildConfig
+  apiVersion: build.openshift.io/v1
+  metadata:
+    namespace: ci
+    name: telco-bastion-cs9
+  spec:
+    successfulBuildsHistoryLimit: 1
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: telco-bastion-cs9:latest
+    source:
+      dockerfile: |
+        FROM registry.ci.openshift.org/openshift/centos:stream9
+        ENV GOCACHE=/go/.cache \
+            GOARM=5 \
+            GOPATH=/go \
+            LOGNAME=deadbeef
+        ENV GOROOT=/usr/local/1.20/go
+        ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+        ENV HOME /output
+        RUN dnf install -y ansible-core wget make jq which && \
+            dnf clean all && \
+            wget https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.11.9/openshift-client-linux.tar.gz && \
+            tar -zxvf openshift-client-linux.tar.gz && \
+            mv oc /usr/local/bin/ && \
+            mv kubectl /usr/local/bin/ && \
+            rm -f openshift-client-linux.tar.gz && \
+            mkdir /output && \
+            chmod -R g+rwx /output && \
+            chmod g+w /etc/passwd && \
+            echo 'echo default:x:$(id -u):$(id -g):Default Application User:/output:/sbin/nologin\ >> /etc/passwd' > /output/fix_uid.sh && \
```

Comment:
I wonder if it's still required, but I believe we can find out only when running a real job.